src/main/java/com/enderio/core/client/render/EnderWidget.java

```diff
@@ -57,7 +57,9 @@
   X_BUT(200, 32, 8, 8),
   X_BUT_PRESSED(200, 40, 8, 8),
   X_BUT_HOVER(200, 48, 8, 8),
-  X_BUT_HOVER_PRESSED(200, 56, 8, 8);
+ X_BUT_HOVER_PRESSED(200, 56, 8, 8),
+
+  NEUTRAL_SLOT_BACKGROUND(112, 176, 16, 16), ;
```

Comment:
yes, very explicitly valid. So one does not have to change the line before the one one wants to add.